src/components/HotRepositories.tsx

```diff
@@ -92,6 +94,17 @@ const HotRepositories = ({ user }: HotReposProps): JSX.Element => {
     setHotRepos([...hotRepos]);
   };
 
+  useEffect(() => {
+    fetchRecommendations("myVotes", 1000, user, "").then((data) => {
+      const flatArrayIds = data.map((repo) => repo.id);
+      // hotRepos.forEach((repo) => {
```

Comment:
this commented code should be put back in IMHO

Reply:
Yeah forgot to push that. Doing it now